modules/site-health/admin-site-health.php

```diff
@@ -297,6 +302,36 @@ public function info_languages( $debug_info ) {
 		return $debug_info;
 	}
 
+	/**
+	 * Adds term props data to the info languages array.
+	 *
+	 * @since 3.4
+	 *
+	 * @param array $value The term props data.
+	 * @return array The term props data formatted for the info languages tab.
+	 */
+	protected function get_info_term_props( $value ) {
+		$return_value = array();
+
+		foreach ( $value as $language_taxonomy => $item ) {
+			$language_taxonomy_array = array_fill( 0, count( $item ), $language_taxonomy );
+
+			$keys_with_language_taxonomy = array_map(
+				function ( $key, $language_taxonomy ) {
+					return $language_taxonomy . '/' . $key;
+				},
+				array_keys( $item ),
+				$language_taxonomy_array
+			);
+
+			$value = array_combine( array_values( $keys_with_language_taxonomy ), array_values( $item ) );
```

Comment:
`array_values()` isn't required: `array_combine()` doesn't care about the array keys.
```suggestion
			$value = array_combine( $keys_with_language_taxonomy, $item );
```